components/Blog/blogList.js

```diff
@@ -2,10 +2,10 @@ import React from 'react'
 
 const BlogComponent = ({ content }) =>
   (<div className='blog-component pa4' >
-    <img src={content.imgUrl} />
+    <img src={content.imgUrl} className='mt4 mr4 mb2 ml0' />
```

Comment:
For accessibility purposes, all images need to use the `alt` attribute. When they serve a purpose other than pure decoration, they need a meaningful description. [See how to write good `alt` text](https://www.deque.com/blog/great-alt-text-introduction/). If the image is decorative, an empty string can be used as the `alt` value, but it must always be present.

Reply:
I added shool name in alt tag